src/components/src/ProductDisplayItem/productdisplayitem.main.tsx

```diff
@@ -625,6 +640,12 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
       slidesToScroll: 1,
     };
 
+    const skuArImagesUrl = Config.arKit.skuArImagesUrl
```

Comment:
This is where we can directly append those configs that you had. Should help keep things cleaner and more dynamic for the implementation 😃